src/main/resources/assets/nutrition/lang/pl_PL.lang

```diff
@@ -0,0 +1,22 @@
+# Keys
+key.nutrition=Otwórz GUI Odżywiania
+
+# GUI
+gui.nutrition:nutrition_title=Odżywianie
+gui.nutrition:close=Zamknij
+gui.nutrition:updating=Aktualizowanie...
+
+# Nutrients
+nutrient.nutrition:grain=Zboża
+nutrient.nutrition:fruit=Owoce
+nutrient.nutrition:vegetable=Warzywa
+nutrient.nutrition:protein=Białka
```

Comment:
Białko (na tej samej zasadzie co nabiał, pomimo że są różne "białka" to w kontekście żywieniowym traktujemy je jako jedno, w chemii i biologii jako grupę i wtedy mówimy "Białka")

Przykład z życia codziennego:
![image](https://github.com/GTNewHorizons/Nutrition/assets/81298696/f0c63817-09b4-4cc8-a1a7-c691c0a7caf5)

Reply:
Zrozumiano :)